src/panels/lovelace/editor/card-editor/hui-dialog-edit-card.ts

```diff
@@ -23,18 +24,13 @@ declare global {
   }
 }
 
+@customElement("hui-dialog-edit-card")
 export class HuiDialogEditCard extends LitElement {
-  protected hass?: HomeAssistant;
-  private _params?: EditCardDialogParams;
-  private _cardConfig?: LovelaceCardConfig;
+  @property() protected hass?: HomeAssistant;
 
-  static get properties(): PropertyDeclarations {
-    return {
-      hass: {},
-      _params: {},
-      _cardConfig: {},
-    };
-  }
+  @property() private _params?: EditCardDialogParams;
+
+  @property() private _cardConfig?: LovelaceCardConfig;
 
   constructor() {
     super();
```

Comment:
For the future, instead of doing a `.bind(this)` in the constructor, you can define the methods like this:

```ts
private _cardPicked = (ev) => {
  // do something…
}
```

Arrow functions are always defined to the defining context, which in this case is the class instance 👍